geanypg/src/pinentry.c

```diff
@@ -108,7 +108,7 @@ gpgme_error_t geanypg_passphrase_cb(void * hook,
 
         execvp(*argv, argv);
         /* shouldn't get here */
-        fprintf(stderr, "GeanyPG: %s\n%s\n", _("Could not use pinentry."), strerror(errno));
+        g_error("%s: %s", _("Could not use pinentry."), strerror(errno));
         exit(1); /* kill the child */
```

Comment:
well, here it `exit()`s so it's probably okay-ish, although it's still less nice `exit()` is nicer (less "OMG something funky happened lemme outta here plzz")